src/auth/urls.py

```diff
@@ -67,72 +84,75 @@ async def login_user(
         # clean old sessions after sending the response
         background_tasks.add_task(crud.task_clean_expired_user_sessions, db_session)
 
-        response = RedirectResponse(next_url)
+        response = RedirectResponse(CURRENT_ENDPOINT + path + "#" + fragment)
         response.set_cookie(
             key="session_id", value=session_id
         )  # this overwrites any past, possibly invalid, session_id
         return response
 
 
-# TODO: deprecate this when possible
 @router.get(
-    "/check",
-    description="Check if the current user is logged in based on session_id from cookies",
+    "/logout",
+    description="Logs out the current user by invalidating the session_id cookie",
 )
-async def check_authentication(
-    request: Request,  # NOTE: these are the request headers
+async def logout_user(
+    request: Request,
     db_session: database.DBSession,
 ):
     session_id = request.cookies.get("session_id", None)
 
     if session_id:
-        await crud.task_clean_expired_user_sessions(db_session)
-        response_dict = await crud.check_user_session(db_session, session_id)
+        await crud.remove_user_session(db_session, session_id)
+        await db_session.commit()
+        response_dict = {"message": "logout successful"}
     else:
-        response_dict = {"is_valid": False}
+        response_dict = {"message": "user was not logged in"}
 
-    return JSONResponse(response_dict)
+    response = JSONResponse(response_dict)
+    response.delete_cookie(key="session_id")
+    return response
 
 
 @router.get(
-    "/info",
+    "/get_user",
```

Comment:
in order for the api to be more RESTful, we should do something like `GET /api/auth/user` rather than `GET /api/auth/get_user`. `/api/auth/user_info` could also make sense

Reply:
changed!
gonna do:

GET /api/auth/user
PATCH /api/auth/user